AI/BattleAI/BattleExchangeVariant.h

```diff
@@ -16,8 +16,8 @@
 
 struct AttackerValue
 {
-	int64_t value;
-	bool isRetalitated;
+	int64_t value{0};
```

Comment:
@IvanSavenko yes, it is clang-tidy change, if it sees than this value is initialized to identical value in all constructors, it moves this value to default. It can be turned off, if desired.

Reply:
@IvanSavenko clang-tidy made this struct as trivially constructible, which is good AFAIK.